src/lib/common/utils.ts

```diff
@@ -0,0 +1,7 @@
+export function isStringTruthy(str: string): boolean {
+  if (str == null) {
+    return false;
+  }
+
+  return !['false', '0', ''].includes(str.toLowerCase());
```

Comment:
Now that the `undefined`s, `null`s, and empty strings have been ruled out by the above if-statement, we only need to check if the payload matches `"false"` and `"0"` (since they would otherwise be classified as truthy).
```suggestion
  return !['0', 'false'].includes(payload.trim().toLowerCase());
```

Reply:
Thanks for your input Ethan, really appreciate it! In this case, if we have checked for truthiness with `if (!payload)` above, would we still need to check for `0` and `false` ? If believe the check also rules out 0, false and other falsy values in JS.

Reply:
Of course; thank you for contributing!

The `if (!payload)` above only catches the `0` literal and `false` literal values. If `payload` happens to be a string value (`"0"` or `"false"`) we need to manually check `"0"` and `"false"` to determine how we want to discern the truthiness of `payload`.

I hope I am understanding your question correctly 😄

Reply:
Yes this makes total sense, I will go ahead and resolve this now, thanks!